src/parachain/vaults.ts

```diff
@@ -316,6 +316,12 @@ export interface VaultsAPI {
         vaultCollateralIdLiteral: CollateralIdLiteral,
         governanceCurrencyIdLiteral: GovernanceIdLiteral
     ): Promise<MonetaryAmount<Currency<GovernanceUnit>, GovernanceUnit>>;
+    /**
+     * Enables or disables issue requests for given vault
+     * @param vaultAccountId The vault ID whose issuing will be toggled
+     * @param acceptNewIssues Boolean denoting whether issuing should be enabled or not
+     */
+    toggleIssueRequests(vaultAccountId: InterbtcPrimitivesVaultId, acceptNewIssues: boolean): Promise<void>;
```

Comment:
the name of the parameter should be `vaultId` rather than `vaultAccountId`. VaultId is the tuple that includes the accountId and the currencyPair

Reply:
Ah, yes, I forgot to change the parameter name in the interface. Thanks!